server/runtime/src/main/java/org/infinispan/server/Extensions.java

```diff
@@ -84,6 +84,7 @@ private void loadServerTasks(ClassLoader classLoader) {
    }
 
    public TaskEngine getServerTaskEngine(EmbeddedCacheManager cm) {
+      // Create marshallers for all tasks
```

Comment:
Is this an old comment? It doesn't seem to belong here anymore.

Reply:
Removed